mu4/notation/internal/notationinteraction.cpp

```diff
@@ -1682,6 +1684,44 @@ void NotationInteraction::drawTextEditMode(QPainter* painter)
     m_textEditData.element->drawEditMode(painter, m_textEditData);
 }
 
+void NotationInteraction::drawSelectionRange(QPainter* painter)
+{
+    if (!m_selection->isRange()) {
+        return;
+    }
+
+    painter->setBrush(Qt::NoBrush);
+
+    QPen pen;
+    pen.setColor(MScore::selectColor[0]);
+    pen.setWidthF(2.0 / painter->worldTransform().toAffine().m11());
+    pen.setStyle(Qt::SolidLine);
+    painter->setPen(pen);
+
+    INotationSelection::RangeBoundingArea rangeData = m_selection->rangeBoundingArea();
+    for (const QRectF& section: rangeData.sections) {
+        QPen pen;
+        pen.setColor(MScore::selectColor[0]);
+        pen.setWidthF(2.0 / painter->worldTransform().toAffine().m11());
+        pen.setStyle(Qt::SolidLine);
+        painter->setPen(pen);
+
+        QPainterPath path;
+        path.addRoundedRect(section, 6, 6);
+
+        QColor fillColor = MScore::selectColor[0];
+        fillColor.setAlpha(10);
```

Comment:
Please, add a method to the configuration (or named constant)

Reply:
I think this is excessive because it is used only here.